### PR TITLE
Assignment Updates: UI for assign and assigned buttons

### PR DESCRIPTION
**Pull request by Erin007**
# Description
We're implementing a handful of changes as outlined in the linked spec below to clarify for users which sections are assigned to which units. Part of these changes includes new buttons to assign and unassign a unit to a given section as shown: 
Screen Shot 2019-10-04 at 10 55 29 PM

This PR includes the UI for these new buttons. 

The `AssignedButton` component toggles between "Assigned" and "Unassign" based on hover state: 
assign-unassign-button

The `AssignButton` will be used by teachers to assign currently unassigned units to a section:
Screen Shot 2019-10-04 at 10 25 29 PM

To make the `AssignedButton` I extended the color options for the `Button` component and tidied the border styles along the way to fix this warning: 
Screen Shot 2019-10-04 at 10 18 45 PM

## Links

- [spec](https://docs.google.com/document/d/1vJySqyKozUxknkhY1KyQXWzbV-0c1vdnpKvRsAsVrdI/edit)
- [jira, LP-705](https://codedotorg.atlassian.net/browse/LP-705) partial

## Testing story

Each of the new components has a Story to ensure that it's rendering as expected. Once the components are used with real data, I will add tests for them in context. 

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by dmcavoy — APPROVED** [review]
This looks good to me Erin. Just one wonder: Why not have one element that handles assigned and assign?

**Comment by Erin007** [comment]
My plan is to have a wrapper component that will know whether to render the orange assign button or the green assigned button, but the only way I could think of to collapse all three states into one button would've meant overhauling a lot of button styling.

File: apps/i18n/common/en_us.json [hunk 1]
```diff
@@ -83,7 +83,9 @@
   "assignCourse": "Assign Course",
   "assignConfirm": "Are you sure you want to assign \"{assignmentName}\" to \"{sectionName}\"?",
   "assignHiddenUnitConfirm": "It looks like you previously hid \"{assignmentName}\" from section \"{sectionName}\". Do you still want to assign this unit and make it visible?",
+  "assigned": "Assigned",
```
Comment (Hamms) [inline apps/i18n/common/en_us.json:86]: looks like this string already existed on line 81

Comment (Hamms) [reply apps/i18n/common/en_us.json:86]: oh, hah; looks like it was actually added in https://github.com/code-dot-org/code-dot-org/pull/31071

**Pull request merged**